core/src/main/java/io/kestra/core/runners/pebble/filters/CondenseStringFilter.java

```diff
@@ -0,0 +1,24 @@
+package io.kestra.core.runners.pebble.filters;
+
+import io.pebbletemplates.pebble.error.PebbleException;
+import io.pebbletemplates.pebble.extension.Filter;
+import io.pebbletemplates.pebble.template.EvaluationContext;
+import io.pebbletemplates.pebble.template.PebbleTemplate;
+
+import java.util.List;
+import java.util.Map;
+
+public class CondenseStringFilter implements Filter {
+
+    @Override
+    public Object apply(Object input, Map<String, Object> args, PebbleTemplate self, EvaluationContext context, int lineNumber) throws PebbleException {
+        if (input instanceof String) return ((String) input).replace("\n", "");
```

Comment:
```suggestion
        if (input instanceof String str) {
            return str.replace("\n", "");
        }
```

Please, always use brackets to facilitate reading and refactoring.
Also pattern matching avoid a cast.